File: majorProject/src/main/java/com/aocc/framework/Game.java
package com.aocc.framework;

//THE FOLLOWING CODE IS SOURCED FROM THE KILOBOLT ANDROID FRAMEWORK

public interface Game {

    public Audio getAudio();

    public Input getInput();

    public FileIO getFileIO();

    public Graphics getGraphics();

    public void setScreen(Screen screen);

    public Screen getCurrentScreen();

    public Screen getInitScreen();
}

File: majorProject/src/main/kotlin/com/aocc/framework/Game.kt
package com.aocc.framework

//THE FOLLOWING CODE IS SOURCED FROM THE KILOBOLT ANDROID FRAMEWORK

interface Game {
    val audio: Audio

    val input: Input

    val fileIO: FileIO

    val graphics: Graphics

    fun setScreen(screen: Screen)

    val currentScreen: Screen

    val initScreen: Screen
}
